### PR TITLE
Update SCSS template directory structure

### PR DESCRIPTION
**Pull request by travis-r6s**
At the moment, when invoking with the scss option, it creates an `assets/scss/...` directory in the root of a project, whereas it should be `src/assets/scss/...`.

Have also taken the liberty of updating the README to change any filepaths from the `dev branch to `master` branch.

### CODE REVIEW

**Comment by travis-r6s** [comment]
And apologies for the earlier closed PR.

**Comment by travis-r6s** [comment]
And like @SharpSeeEr , this is my first attempt at creating/editing a vue cli plugin, so I presume this is the needed fix, but am not sure how to test it.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 [Vue CLI 3.x](https://github.com/vuejs/vue-cli) plugin to add buefy to your Vue Project
 
-> Well, apparently you're an [adventurer](https://github.com/vuejs/vue-cli/blob/dev/README.md#status-beta). We are happy to see you here.
```
Comment (anteriovieira) [inline README.md:13]: `dev` is the branch for `@vue/cli 3.0`.

**Comment by anteriovieira** [comment]
Reopened in https://github.com/buefy/vue-cli-plugin-buefy/pull/6

**Pull request closed without merging**